includes/user-functions.php

```diff
@@ -593,3 +593,48 @@ function give_new_user_notification( $user_id = 0, $user_data = array() ) {
 }
 
 add_action( 'give_insert_user', 'give_new_user_notification', 10, 2 );
+
+
+/**
+ * Get Donor Name By
+ *
+ * Retrieves the donor name based on the id and the name of the user or donation
+ *
+ * @access      public
+ * @since       1.0
+ *
+ * @param       int     $id     The ID of donation or donor
+ * @param       string  $from   From will be a string to be passed as donation or donor
+ *
+ * @return      array
```

Comment:
@ravinderk This is a minor issue which is missed. However, i'll make sure that this will be double checked by me next time submitting a PR.